master/usage/calicoctl.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing and Configuring calicoctl
+---
+
+This document outlines how to install and configure calicoctl which is the
+primary tool for viewing, manipulating, and creating Calico objects on the
+command line.
+
+### Location, location, location
+
+The majority of calicoctl operations only need access to the datastore that
+Calico is using.  This means that the tool can be ran from any host 
+or container that can reach the datastore and has the proper credentials for
+access.
+
+#### Installing calicoctl
+
+The calicoctl tool can be downloaded from the
+[release page of the calicoctl repository]({{site.data.versions[page.version].first.components.calicoctl.url}}),
+set executable, and then it is ready to run.
+
+```
+curl -O -L {{site.data.versions[page.version].first.components.calicoctl.download_url}}
+chmod +x calicoctl
+```
+
+> *Note:* Move calicoctl to a directory in your PATH or add the directory
```

Comment:
because personally I do not like putting binaries that I am unfamiliar with in my PATH, plus it is not necessary for operation.  I guess there is also the possibility that what ever directory used might not actually be in a users PATH.